week1/main.js

```diff
@@ -0,0 +1,174 @@
+editedPrice.forEach((price) => {
+  const removedComma = price.innerText.slice(0).replace(/\D/g, "");
+  priceList[price.previousElementSibling.innerText] = Number(removedComma);
+});
+
+// let priceList = {
+//   "리코타 치킨 버거": 6300,
+//   "더블 빅맥": 7300,
+//   "맥치킨 모짜렐라": 5600,
+//   "에그 불고기 버거": 4000,
+//   "베이컨 토마토 디럭스": 6300,
+//   "더블 치즈버거": 5200,
+//   슈슈버거: 5300,
+//   슈비버거: 6300,
+//   치즈버거: 3100,
+//   "더블 치즈버거": 5200,
+//   "쿼터파운드 치즈": 6000,
+//   "불고기 버거": 3100,
+// };
+
+function chooseMenu() {
+  const burgerCard = document.querySelectorAll(".burger__card");
+  burgerCard.forEach((burger) => {
+    burger.addEventListener("click", (e) => {
+      addToCart(e);
+    });
+  });
+}
+
+function addToCart(e) {
+  const currentBurger = e.currentTarget.querySelector(
+    ".burger__card__detail__name"
+  );
+  if (currentBurger.innerText in addList) {
+    continueAdd(e, currentBurger);
+  } else {
+    initAdd(e, currentBurger);
+  }
+}
+
+function initAdd(e, currentBurger) {
+  addList[currentBurger.innerText] = [
+    1,
+    `₩${priceList[currentBurger.innerText]}`,
+  ];
+
+  const burgerCart = $(".cart__menu");
+  const menuContainer = document.createElement("div");
+  const menuText = document.createElement("div");
+  const countText = document.createElement("input");
+  const priceText = document.createElement("div");
+  const delBtn = document.createElement("button");
+
+  burgerCart.appendChild(menuContainer);
+  menuContainer.appendChild(menuText);
+  menuContainer.appendChild(countText);
+  menuContainer.appendChild(priceText);
+  menuContainer.appendChild(delBtn);
+
+  menuContainer.classList.add("add__menu");
+  menuText.classList.add("add__menu__text");
+  countText.classList.add("add__menu__count");
+  countText.id = `${currentBurger.innerText}`;
+  priceText.classList.add("add__menu__price");
+  delBtn.classList.add("delete__btn");
+
+  menuText.textContent = currentBurger.innerText;
+  countText.value = addList[currentBurger.innerText][0];
+  priceText.textContent = addList[currentBurger.innerText][1];
+  delBtn.textContent = "X";
+  countText.type = "number";
+
+  addAmount(currentBurger);
+  clickCancelBtn();
+  checkInput(currentBurger);
+}
+
+function checkInput(currentBurger) {
+  const addCount = document.getElementById(currentBurger.innerText);
+  addCount.addEventListener("click", (e) => {
+    const changedInput = Number(addCount.value);
+    amountPrice +=
+      (changedInput - addList[currentBurger.innerText][0]) *
+      priceList[currentBurger.innerText];
+    amountText.textContent = `₩${amountPrice.toLocaleString()}`;
+    addList[currentBurger.innerText][0] = changedInput;
+  });
+}
+
+function continueAdd(e, currentBurger) {
+  continueCount(currentBurger);
+  addAmount(currentBurger);
+}
+
+function continueCount(currentBurger) {
+  const addCount = document.getElementById(currentBurger.innerText);
+  addCount.value = Number(addCount.value) + 1;
```

Comment:
음 이게 다른 코드랑 어떻게 작용할 지 잘 모르겠지만 
여기서는 카운트만 올려주면 되니까
```suggestion
  document.getElementById(currentBurger.innerText).value++;
```
이렇게 해도 될거같은데 문법 에러가 나는지는 모르겠당 나는 아래처럼 작성했어!
$(`.cart__list-${burgerName}--amount`).value++;